Cargo.toml

```diff
@@ -25,16 +25,14 @@ chrono = { version = "0.4", features = ["serde"] }
 clap = "2.33"
 http = "0.1.18"
 maplit = "1.0"
-reqwest = "0.9"
-serde = "1.0"
-serde_derive = "1.0"
+reqwest = { version = "0.10.0-alpha.1", features = ["json", "blocking"] }
```

Comment:
Why not wait till it's stable?

Reply:
I believe the unstable part is the new async features. The entire synchronous API is the same, it's just been moved to `reqwest::blocking`.